### ISSUE TITLE
[Accessbility] Screen reader is not announcing tooltip on focus/hover of "Search Workspace" button

### ISSUE THREAD

**Issue by essjay05** [microsoft/vscode#267817]
# PROBLEM:
The screen reader is not announcing the value of the text that appears in the tooltip that appears on focus/hover of the "Search Workspace" button in the `.monaco-action-bar`. It only announces the text value of the button itself.

## OBSERVED RESULTS:
When the "<Search icon> Workspace" button in the `.monaco-action-bar` is focused or hovered when the screen reader is enabled, the screen reader does NOT announce the text of the tooltip that pops up "Search Workspace ---- Welcome - Workspace - Visual Studio Code". 
* In **Chromebook + ChromeVox** it is reading "Workspace Button Toolbar, Press search plus space to activate." 
* This also is the case with a **Mac + VoiceOver,** it reads "Workspace Button, You are currently on a toolbar, inside of a toolbar."

## EXPECTED RESULTS:
On focus/hover of the "Search Workspace" button in the `.monaco-action-bar`, the text value of the tooltip that appears should be announced by the screen reader for screen reader users.

## USER IMPACT:
Ensure that additional content that appears and disappears in response to a hover state is perceivable and supports certain interaction requirements.

## STEPS TO REPRODUCE:
1. Go to [VSCode Dev](https://vscode.dev/)
2. Enable your screen reader.
3. Focus on hover on the "Search Workspace" button in the `.monaco-action-bar`/toolbar at the top of VSCode.
4. Observe the tooltip text that appears on focus/hover and that there is no screen reader announcement of that tooltip text.

## ENVIRONMENT DETAILS:
* Machine: Chromebook
Browser: Version 136.0.7103.158 (Official Build) (64-bit)
Accessibility Tool: ChromeVox.

* Machine: MacBook Pro
Browser: Version 137.0.7151.104 (Official Build) (arm64)
Accessibility Tool: VoiceOver.

Version: 1.104.1
Commit: 0f0d87fa9e96c856c5212fc86db137ac0d783365
User Agent: Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36
Embedder: vscode.dev
